src/object-mirrors.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var ERROR_SOURCE_PROPERTY_NOT_IMPLEMENTED = 'Requiring undefined property on source object';
```

Comment:
but we will do this simply by wrapping the call to xhrSetup in a try-catch and propagating a custom error from there ...  see other PR